spec/unit/resources/server_profile_template/from_sp_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './../../../spec_helper'
+
+describe 'oneview_test::server_profile_template_from_sp' do
```

Comment:
Suggestion: If this test is just API500, it might be better to be oneview_test_api500_synergy.